package ssp;

message Test {
    optional int32 a = 1;
    optional int32 b = 2;
    optional int32 c = 3;
    optional string d = 4;
};

message TestList {
    optional int32 a = 1;
    repeated Test tl = 2; //数组
};

message UserInfoBase {
    optional uint32 ver = 1; // 版本号
    optional uint32 user_id = 2;
    optional string user_name = 3;
    optional string password = 4;
    optional uint32 max_t = 33;
};
